docs/source/plotting/index.rst

```diff
@@ -476,7 +476,7 @@ Custom Colormap (MantidWorkbench)
 You can view the premade Colormaps `here <https://matplotlib.org/2.2.3/gallery/color/colormap_reference.html?highlight=colormap>`_.
 These Colormaps can be registered and remain for the current session, but need to be rerun if Mantid has been reopened. Choose the location to Save your Colormap file wisely, outside of your MantidInstall folder!
 
-The following methods show how to Load, Convert from MantidPlot format, Create from Scratch and Visualise a Custom Colormap.
```

Comment:
I think this actually do need to say MantidPlot. @DanielMurphy22 Am I right in saying these were about converting from the old custom format to matplotlib?

Reply:
Yes indeed this is for converting from a MantidPlot colormap to one that can be loaded for Workbench